qml/MainScreen.qml

```diff
@@ -217,6 +227,56 @@ Item {
         defaultValue: "file:///usr/share/asteroid-launcher/applauncher/000-default-horizontal.qml"
     }
 
+    ConfigurationValue {
+        id: watchfaceNightstandSource
+        key: "/desktop/asteroid/nightstand/watchface"
+        defaultValue: "file:///usr/share/asteroid-launcher/watchfaces/005-analog-nordic.qml"
+    }
+
+    ConfigurationValue {
+        id: nightstandBrightness
+        key: "/desktop/asteroid/nightstand/brightness"
+        defaultValue: 30
+    }
+
+    ConfigurationValue {
+        id: nightstandDelay
+        key: "/desktop/asteroid/nightstand/delay"
+        defaultValue: 5
```

Comment:
Would you mind documenting above what's the purpose of this function (not what it does but more, why you felt the need to introduce it) and why you came up with 5 seconds as a default value?

Spontaneously I seem to think of it as a workaround for bad connections and 5 seconds seems a bit high, maybe 2 or 3 seconds would have been enough but maybe I'm missing something ? Because I imagine it must feel very "laggy" if disconnecting the watch from its charger keeps this on for 5 seconds

Reply:
If it's 3AM and I grab the watch in the dark without my glasses on to take a look, I don't want it to immediately switch to daytime watchface, but maintain the more readable nightstand watchface for a few seconds.

Reply:
See also https://github.com/AsteroidOS/asteroid/issues/151#issuecomment-1204035843 for an elaboration of that scenario and corresponding requirements.